### ISSUE TITLE
Address S1104: Fields should not have public accessibility

### ISSUE THREAD

**Issue by bart-vmware** [SteeltoeOSS/Steeltoe#914]
Address existing violations of [S1104: Fields should not have public accessibility](https://rules.sonarsource.com/csharp/RSPEC-1104) in the codebase and set severity to `Warning` in `Steeltoe.Debug.ruleset` and `Steeltoe.Release.ruleset`.

To find existing violations, enable the rule (see above) and rebuild `src/Steeltoe.All.sln` to make them appear in the Output window.

To address the violations, choose from the following on a case-by-case basis:
- Make the field private
- Convert the field into an auto-property (consider a get-only auto-property)
- Make the field `const` or `readonly`
- ~Suppress the violation in code using `#pragma warning disable/restore`, preceded by a justification comment if not obvious~

**Comment by bart-vmware** [comment]
Closing in favor of #928.
